gulp/sass.js

```diff
@@ -96,7 +96,7 @@ module.exports = (gulp, plugins, blueprint) => {
             // package the variables list for consumption -- no imports or functions
             .pipe(plugins.stripCssComments())
             .pipe(plugins.replace(/\n{3,}/g, "\n\n"))
-            .pipe(plugins.replace(/(@import|\/\/).*\n+/g, ""))
+            .pipe(plugins.replace(/(@import|\/\/).*\r\n+/g, ""))
```

Comment:
maybe we can actually remove the `\n` from this expression entirely and instead capture it in the `\n{3,}` rule? would be better to handle newlines in fewer places.